### DISCUSSION TITLE
[BUG] wallet status not synced

### DISCUSSION THREAD

**Discussion by greenranvnalo** [Chia-Network/chia-blockchain#3971] [Support]
**Describe the bug**
wallet status is says not synced even tho the node says synced

( im new to chia, and i didnt start farming or plotting just yet )

r

**Expected behavior**
i expect it to be synced

**Screenshots**
![Untitled](https://user-images.githubusercontent.com/31481669/117302814-562e8880-ae7c-11eb-9be9-33929167945e.png)
![walllet](https://user-images.githubusercontent.com/31481669/117302822-59c20f80-ae7c-11eb-881c-22efe8888b20.png)

**Desktop 

- OS: windows 10 pro
- OS Version/Flavor: windows 10 pro 20h2
- CPU: ryzen 5900x

**Comment by voipcall** [comment]
same problem

**Comment by michfen** [comment]
same here

**Comment by ervin20** [comment]
Same problem

**Comment by gillopillo** [comment]
same here. This is insanely frustrating. It says farming but I am not sure what's going on if my wallet can't sync.

**Comment by Helheimbaby** [comment]
You have to Portforward your Router TCP & UDP 8444 Look at following Link for Questions 
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#why-does-my-node-have-no-connections-how-can-i-get-more-connections

**Reply by gillopillo** [reply]
I port forwarded my router, and disabled upnp on both machines in the very beginning. This is happening anyways, on both machines. Port forwarding can't solve everything.

**Reply by Helheimbaby** [reply]
Iam sorry it doesn´t helps to solve your Problem, in my Case it does help to sync the Wallet.

**Comment by kshyshkin** [comment]
same problem, get some strange logs about this problem  https://github.com/Chia-Network/chia-blockchain/discussions/4519

**Comment by DMS-1981** [comment]
same problem, lost coin. Any help

**Comment by howardnguyen714** [comment]
I have the same issue as well. Already set up port forwarding too. Help appreciated. Thanks!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
